src/annotator/index.js

```diff
@@ -1,15 +1,6 @@
 'use strict';
 
 const configFrom = require('./config/index');
-require('../shared/polyfills');
-
-// Polyfills
-
-// `document.evaluate` polyfill for IE 11.
-if (!window.document.evaluate) {
-  const wgxpath = require('wicked-good-xpath');
-  wgxpath.install();
-}
```

Comment:
This polyfill is now loaded by the boot script in `src/boot/boot.js`, if needed, before this code runs.